parselglossy/read_yaml.py

```diff
@@ -1,11 +1,39 @@
-import yaml
-from typing import Any, Dict
+#
+# parselglossy -- Generic input parsing library, speaking in tongues
+# Copyright (C) 2019 Roberto Di Remigio, Radovan Bast, and contributors.
+#
+# This file is part of parselglossy.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+# For information on the complete list of contributors to the
+# parselglossy library, see: <http://parselglossy.readthedocs.io/>
+#
+
+from pathlib import Path
 
+import yaml
 
-JSONDict = Dict[str, Any]
+from .utils import JSONDict
```

Comment:
very good to collect these in a file - perhaps we should call the file `types` - but we can do this later.